hokusai/cli/review_app.py

```diff
@@ -0,0 +1,35 @@
+import click
+from shutil import copyfile
+import yaml
+
+import hokusai
+
+from hokusai.cli.staging import staging, KUBE_CONTEXT
+from hokusai.lib.common import set_verbosity, CONTEXT_SETTINGS
+from hokusai.lib.config import config
+
+@staging.group()
+def review_app(context_settings=CONTEXT_SETTINGS):
+  """Manage/Create review apps"""
+  pass
+
+@review_app.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('app_name', type=click.STRING)
+@click.option('-v', '--verbose', type=click.BOOL, is_flag=True, help='Verbose output')
+def create(app_name, verbose):
+  set_verbosity(verbose)
+  # copy staging yml file
+  create_new_app_yaml('hokusai/staging.yml', app_name)
+
+def create_new_app_yaml(original_yaml_file, app_name, current_namespace='default'):
+  with open(original_yaml_file, 'r') as stream:
+    try:
+      yaml_content = list(yaml.load_all(stream))
+      # @TODO: replace namespace: default with new namespace name
```

Comment:
I would not make any assumptions that the file configures their app in the `"default"` namespace - it coul dbe any namespace.  So will need to traverse keys and replace any `namespace` keys with the new namespace

Reply:
Maybe need to dig into the Kubernetes object specs a bit more here but AFAIK _all_ objects support the key `metadata.namespace` so it might just be enough to iterate through all top-level objects defined in the source yml and simply overwrite this key